Dockerfile

```diff
@@ -1,34 +1,71 @@
-FROM redis:latest as builder
+# BUILD redisfab/redisgraph:${VERSION}-${ARCH}-${OSNICK}
 
-ENV DEPS "automake peg libtool autoconf python python-setuptools python-pip wget build-essential cmake m4 libgomp1"
+ARG REDIS_VER=6.0.1
+
+# OSNICK=bionic|stretch|buster
+ARG OSNICK=buster
+
+# OS=debian:buster-slim|debian:stretch-slim|ubuntu:bionic
+ARG OS=debian:buster-slim
+
+# ARCH=x64|arm64v8|arm32v7
+ARG ARCH=x64
+
+#----------------------------------------------------------------------------------------------
+FROM redisfab/redis:${REDIS_VER}-${ARCH}-${OSNICK} AS redis
```

Comment:
Please try switching to https://hub.docker.com/_/redis

Reply:
OK, just recalled the story behind Redis docker:
The official repo uses https://github.com/docker-library/redis to create their Redis image. We are based on the same repo, with our own tweaks. For instance, we have a scheme that supports other platforms than debian:buster and alpine.
So, in the bottom line, I think we better stay with our version of Redis docker.